tests/test_cli.py

```diff
@@ -189,6 +197,9 @@ def test_pipeline_ae(self, config, callback_values):
         config.ae.userid_column_name = "user_col"
         config.ae.userid_filter = "user321"
 
+        expected_columns = self._read_data_file(os.path.join(TEST_DIRS.data_dir, 'columns_ae.txt'))
```

Comment:
will we (most) always be loading from TEST_DIRS? If so, maybe we should move the path join in to `_read_data_file`.

Reply:
I would really rather not, there are other dirs that unittests interact with like the `tests/expected_data` dir and the `tmp_path` so I wouldn't want to hinder the method, potentially making it less flexible in the future.